app/src/org/commcare/activities/AppManagerActivity.java

```diff
@@ -53,7 +55,10 @@ protected void onResume() {
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
         super.onCreateOptionsMenu(menu);
-        menu.add(0, MENU_CONNECTION_DIAGNOSTIC, 0, Localization.get("home.menu.connection.diagnostic")).setIcon(android.R.drawable.ic_menu_preferences);
+        menu.add(0, MENU_ADVANCED_SETTINGS, 0, Localization.get("app.manager.advanced.settings.option"))
+                .setIcon(android.R.drawable.ic_menu_preferences);
+        menu.add(0, MENU_CONNECTION_DIAGNOSTIC, 1, Localization.get("home.menu.connection.diagnostic"))
+                .setIcon(android.R.drawable.ic_menu_preferences);
```

Comment:
Can you change `ic_menu_preferences` to be something different, so on early Android versions the user doesn't see the same icon for both these menu entries?